src/components/landing/Introduction.jsx

```diff
@@ -0,0 +1,51 @@
+import Image from "next/image";
+import circuitImage from "@/public/circuit.webp";
+import { TiArrowSortedDown } from "react-icons/ti";
+
+const Introduction = () => {
+  return (
+    <div className="pt-8 flex items-stretch justify-between bg-white brightness-95">
+      <Image
+        src={circuitImage}
+        alt="Left Circuit Image"
+        className="w-1/4 mr-5"
+      />
+
+      <div className="w-1/2 flex flex-col">
+        <div className="flex flex-grow flex-col items-center justify-center">
+          <p className="whitespace-nowrap text-2xl xs:text-4xl sm:text-5xl md:text-6xl lg:text-7xl xl:text-8xl text-black">
+            AI @ UCR
+          </p>
+          <p className="whitespace-nowrap text-3xs xs:text-2xs sm:text-sm md:text-lg lg:text-xl xl:text-2xl text-black">
+            Empowering the Future of Tech
+          </p>
+        </div>
+
+        <div className="flex flex-grow items-center justify-center">
+          <button className="relative w-6 h-6 xs:w-10 xs:h-10 sm:w-14 sm:h-14 md:w-16 md:h-16 lg:w-20 lg:h-20 xl:w-28 xl:h-28 bg-ai-blue-500 rounded-full text-white">
```

Comment:
do not use pixel values for heights and widths. in this case padding will be a better solution